webapp/channels/src/components/admin_console/elasticsearch_settings.tsx

```diff
@@ -97,7 +121,7 @@ export default class ElasticsearchSettings extends AdminSettings {
         return this.state.canSave;
     };
 
-    doTestConfig = (success, error) => {
+    doTestConfig = (success: (data?: any) => void, error: (error: any) => void): void => {
```

Comment:
We can remove the parameter for `data` here since it isn't used.

Also `error` can be `string` instead of `any`.